### DISCUSSION TITLE
v1.0.20240102beta has dropped support for custom names?

### DISCUSSION THREAD

**Discussion by serveroutlet** [Louisvdw/dbus-serialbattery#931] [General]
Hello everyone,

Maybe you can help me: 
https://github.com/Louisvdw/dbus-serialbattery/tree/v1.0.20231125beta
So this version has an issue it is doubling the JBD stated SOC capacity when compared to the xiaoxing app. However it has the option for custom names including naming the temperature probes.

https://github.com/Louisvdw/dbus-serialbattery/tree/v1.0.20240102beta
This version reports the correct SOC (matches the xiaoxing app) but does not support custom names the options are actually missing form the ini file completely.

Is there a version that has both these features in 1 driver?

Thank you for your time.

Thanks

**Comment by mr-manuel** [comment]
Just read the changelog.
